app/models/sections/info/movementScenario/MovementScenario.scala

```diff
@@ -230,16 +217,63 @@ object MovementScenario extends Enumerable.Implicits with Logging {
 
   }
 
+  /**
+   * emcs: certified_consignee / import_for_certified_consignee
+   */
+  case object CertifiedConsignee extends WithName("certifiedConsignee") with MovementScenario {
+
+    override def destinationType: DestinationType = DestinationType.CertifiedConsignee
+
+    override def movementType(implicit request: DataRequest[_]): MovementType = request.isCertifiedConsignor match {
+      case true => MovementType.UkToEu
+      case false =>
+        logger.error(s"[movementType] invalid UserType for COD journey: ${request.userTypeFromErn}")
+        throw InvalidUserTypeException(s"[MovementScenario][movementType] invalid UserType for COD journey: ${request.userTypeFromErn}")
+    }
+
+    override val stringValue: String = "certified consignee"
+  }
+
+  /**
+   * emcs: temp_certified_consignee / import_for_temp_certified_consignee
+   */
+  case object TemporaryCertifiedConsignee extends WithName("temporaryCertifiedConsignee") with MovementScenario {
+
+    override def destinationType: DestinationType = DestinationType.TemporaryCertifiedConsignee
+
+    override def movementType(implicit request: DataRequest[_]): MovementType = request.isCertifiedConsignor match {
+      case true => MovementType.UkToEu
+      case false =>
+        logger.error(s"[movementType] invalid UserType for COD journey: ${request.userTypeFromErn}")
+        throw InvalidUserTypeException(s"[MovementScenario][movementType] invalid UserType for COD journey: ${request.userTypeFromErn}")
+    }
+
+    override val stringValue: String = "temporary certified consignee"
+
+  }
+
+  case object ReturnToThePlaceOfDispatch extends WithName("returnToThePlaceOfDispatch") with MovementScenario {
```

Comment:
This object's destinationType and movementType methods aren't tested in MovementScenarioSpec

Reply:
Will add that in